### PR TITLE
[SOAR-4206, SOAR-4207]Update Jira and abuseipdb to v4 Python plugin runtime

### PR DESCRIPTION
**Pull request by gusingh-r7**
### Description

Describe the proposed changes:

  - Update Jira and Abuseipdb to v4 Python plugin runtime
  - Set cloud_ready to true

### Testing

Any testing information goes here.

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

**Review by tschmidtke-r7 — APPROVED** [review]
lgtm. One request though. In the interest of slowly removely `komand` references, can the `komand_jira` and `komand_abuseipdb` directories be renamed to have `icon_` as the prefix and then regenerate the plugins?

**Review by norenstrakh-r7 — DISMISSED** [review]
LGTM 👍

**Review by jschipp-r7 — DISMISSED** [review]

**Review by zyoutz-r7 — DISMISSED** [review]
Let's get flagging in place and then this looks good 👍🏻

File: abuseipdb/help.md [hunk 1]
```diff
@@ -28,9 +28,7 @@ Example input:
 
 ```
 {
-  "credentials": {
-    "secretKey": "e73h82c63847f3ff1h5216b556edh153h30430d73bchhe680f70h1d8885fb8bb130b46c7767d6886"
-    }
+  "credentials": "e73h82c63847f3ff1h5216b556edh153h30430d73bchhe680f70h1d8885fb8bb130b46c7767d6886"
```
Comment (mrinehart-r7) [inline abuseipdb/help.md:31]: Revert

File: jira/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM komand/python-3-37-slim-plugin:3
+FROM rapid7/insightconnect-python-3-38-plugin:4
```
Comment (mrinehart-r7) [inline jira/Dockerfile:1]: Can you use the slim SDK? IIRC it is 3-38-slim-plugin

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]

File: abuseipdb/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM komand/python-3-37-slim-plugin:3
+FROM rapid7/insightconnect-python-3-38-plugin:4
```
Comment (zyoutz-r7) [inline abuseipdb/Dockerfile:1]: I think Mike caught this on the other one.  Are we able to keep using slim here?

**Review by mrinehart-r7 — DISMISSED** [review]

**Review by zyoutz-r7 — APPROVED** [review]

**Review by tschmidtke-r7 — APPROVED** [review]

**Pull request merged**